Framework/Properties/Mantid.properties.template

```diff
@@ -23,6 +23,8 @@ Q.convention = Inelastic
 
 mantidqt.python_interfaces = ILL/Drill.py Direct/DGS_Reduction.py Direct/DGSPlanner.py Direct/PyChop.py Direct/MSlice.py SANS/ORNL_SANS.py Utility/TofConverter.py Diffraction/Engineering_Diffraction.py Diffraction/Powder_Diffraction_Reduction.py Utility/FilterEvents.py Diffraction/HFIR_4Circle_Reduction.py Utility/QECoverage.py SANS/ISIS_SANS.py Muon/Frequency_Domain_Analysis.py Muon/Elemental_Analysis.py Muon/Frequency_Domain_Analysis_Old.py Muon/Muon_Analysis.py General/Sample_Transmission_Calculator.py
 
+mantidqt.python_interfaces_io_registry = Engineering_Diffraction_register.py
```

Comment:
Could I ask the purpose of this key? Is it not possible just to check for the existence of a file named the same as the startup file but with `_register.py` appended?

Reply:
I originally did it this way to make it slightly easier/clearer to remove if and when the way the interface startups are handled gets changed, as per our conversation a couple of weeks ago. Happy to change it though if you think it's unneccesary

Reply:
As this is a short-term fix I think this is okay. It will cut down on the number of checks for files that don't exist on startup. 👍🏻